lib/config/config.go

```diff
@@ -560,14 +561,32 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 			}
 		}
 
-		if _, err := os.Stat(c.Conmon); err != nil {
-			return errors.Wrapf(err, "invalid conmon path")
+		// Validate the conmon path
+		if err := c.ValidateConmonPath("conmon"); err != nil {
+			return errors.Wrapf(err, "conmon validation")
 		}
 	}
 
 	return nil
 }
 
+// ValidateConmonPath checks if `Conmon` is set within the `RuntimeConfig`.
+// If this is not the case, it tries to find it within the $PATH variable.
+// In any other case, it simply checks if `Conmon` is a valid file.
+func (c *RuntimeConfig) ValidateConmonPath(executable string) error {
+	if c.Conmon == "" {
+		conmon, err := exec.LookPath(executable)
+		if err != nil {
+			return errors.Wrapf(err, "%q not found in $PATH", executable)
+		}
+		c.Conmon = conmon
+		logrus.Debugf("using conmon from $PATH %q", conmon)
+	} else if _, err := os.Stat(c.Conmon); err != nil {
+		return errors.Wrapf(err, "invalid conmon path")
+	}
```

Comment:
I'd prefer to always to a `logrus.Infof(...)` here since forgetting to set conmon explicitly could be a source of bugs/undesired behaviours etc.

Reply:
To be more precise: I'd always log the path of the conmon we're using, independent where it came from.

Reply:
Added a log message in any succeeding case.